packages/components/src/Checkbox/Checkbox.tsx

```diff
@@ -175,6 +175,13 @@ export const Label = ({
 };
 
 /**
+ * ```ts
+ * import {Checkbox} from "@chanzuckerberg/eds-components";
+ * ```
+ * ```ts
+ * import Checkbox, {CheckboxInput, Label} from '@chanzuckerberg/eds-components/lib/Checkbox';
+ * ```
+ *
```

Comment:
Interestingly `Checkbox` and `Tooltip` docstrings don't show up in Storybook (even before this PR). Import paths still added for VSCode hover
![single code blcok](https://user-images.githubusercontent.com/86632227/154617321-20e79de6-df1c-495b-b08c-65ddac8bca05.png)

Reply:
Oh, I think I've hit this issue before. There's a strange bug with regards to `displayName` – try moving the display name lines to the bottom of the file, above the export line, and make sure the main component is last.

Like
```
[checkbox component code]

Checkbox.displayName = "Checkbox";

CheckboxInput.displayName = "CheckboxInput";

export default Checkbox;
```

Reply:
At least for `Tooltip`, just removing the display name seems to fix it. I don't remember why we added that in the first place. For the `Checkbox`, we need it because of the `forwardRef`, but I can only get the `CheckboxInput` docstring to show up on the docs page.

Reply:
Adding `export` in front of `const Checkbox = React.forwardRef<HTMLInputElement, CheckboxProps>(` and moving the checkbox + display name above the checkbox input seems to fix it for me 🤷 
<img width="1191" alt="checkbox in storybook" src="https://user-images.githubusercontent.com/7761701/154747963-3e3549fd-c81d-4e6e-a528-b2fe90bba145.png">

Reply:
Did similarly as suggested, and docstrings are back! thanks